### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by MabuzaM**
- [DEMO LINK](https://MabuzaM.github.io/layout_catalog-en/)
- [TEST REPORT LINK](https://MabuzaM.github.io/layout_catalog-en/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,447 @@
...
+    <main>
+      <div class="page__content">
+        <div class="page__content-product" data-qa="card">
+          <img src="./images/imac.jpeg" alt="iMac PC" class="page__content-product-image">
+
+          <p class="page__content-product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <div class="page__content-product-code">
```
Comment (efdalyalcin) [inline src/index.html:103]: I think it is hard to follow the class names, maybe you can create another block for the for the cards.  You can just name it card and it will be easier to name the modifiers, elements and other stuff of this block.

File: src/styles/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.page__header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  margin: 0 auto;
```
Comment (efdalyalcin) [inline src/styles/blocks/header.scss:3]: I know in this task we didn't look at the header but:
This doesn't do anything since it always stretches to whole page width. I think you should add a max-width so that it there is no huge gap between logo and other stuff in a wider screen. Say 1200px or whatever you want.

File: src/styles/blocks/page-content.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    padding: 50px 40px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 auto;
+
+    &-product {
+      display: flex;
+      flex-direction: column;
+      flex-basis: 200px;
```
Comment (efdalyalcin) [inline src/styles/blocks/page-content.scss]: default value is column anyway, i think there is no need for both of these

File: src/styles/blocks/page-content.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        }
+      }
+
+      &-cta-link {
+        height: 40px;
+        width: 160px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        align-self: center;
```
Comment (efdalyalcin) [inline src/styles/blocks/page-content.scss]: This is also not needed i guess.

File: src/styles/blocks/page-content.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        border-radius: 5px;
+        text-align: center;
+        color: $white-color;
+        font-weight: $large-font-weight;
+        font-size: $medium-font-size;
+        text-transform: uppercase;
+
+        &:hover {
+          background-color: $white-color;
+          color: $primary-color;
```
Comment (efdalyalcin) [inline src/styles/blocks/page-content.scss:96]: I think you should add a transition for this.

File: src/styles/blocks/stars.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    margin-right: 4px;
+  }
+
+  &--1 .stars__star:nth-child(-n+1),
+  &--2 .stars__star:nth-child(-n+2),
+  &--3 .stars__star:nth-child(-n+3),
+  &--4 .stars__star:nth-child(-n+4),
+  &--5 .stars__star:nth-child(-n+5) {
+    background-image: url("/images/star-active.svg");
+  }
```
Comment (efdalyalcin) [inline src/styles/blocks/stars.scss]: you can use a @for loop for this and also `&--1 .stars__star:nth-child(-n+1)` can be coded like `&--1 &:nth-child(-n+1)`

File: src/styles/utils/normalize.scss [hunk 7]
```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: $normal-font-weight;
+  font-size: $normal-font-size;
+  line-height: $normal-line-height;
+  color: var(--text-secondary-color);
```
Comment (efdalyalcin) [inline src/styles/utils/normalize.scss:10]: I think you forgot to change this one :)

**Review by efdalyalcin — CHANGES_REQUESTED** [review]
Well done! 

There are only only slight changes, then you are good to go.

File: src/styles/utils/normalize.scss (hunk 7 above)
Comment (MabuzaM) [reply src/styles/utils/normalize.scss:10]: How? I don't understand

File: src/styles/blocks/stars.scss (hunk 6 above)
Comment (MabuzaM) [reply src/styles/blocks/stars.scss]: Okay thanx done.

File: src/styles/blocks/page-content.scss (hunk 4 above)
Comment (MabuzaM) [reply src/styles/blocks/page-content.scss]: Okay done. thanx

File: src/styles/blocks/page-content.scss (hunk 3 above)
Comment (MabuzaM) [reply src/styles/blocks/page-content.scss]: Noted. fixed

File: src/index.html (hunk 1 above)
Comment (MabuzaM) [reply src/index.html:103]: Noted.

File: src/styles/utils/normalize.scss (hunk 7 above)
Comment (ansu16281) [reply src/styles/utils/normalize.scss:10]: @MabuzaM He probably means that your `color` variable is in the CSS format, but not SCSS. But I don't think it's a mistake tbh.

**Review by meforoma — APPROVED** [review]
